src/sentry_backend.h

```diff
@@ -6,10 +6,12 @@
 #include "sentry_scope.h"
 
 struct sentry_backend_s;
+struct sentry_ucontext_s;
```

Comment:
doesn’t the `sentry_boot.h` above pull in `sentry_ucontext_t` already?

Reply:
Yes it does :) I'll remove this